_posts/2025-06-24-kafka-queue.md

```diff
@@ -0,0 +1,416 @@
+---
+layout: post
+title:  "Using Kafka Queue feature with Strimzi"
+date: 2025-06-24
+author: tina_selenge
+---
+
+The Kafka Queue feature introduced by [KIP-932](https://cwiki.apache.org/confluence/display/KAFKA/KIP-932%3A+Queues+for+Kafka#KIP932:QueuesforKafka) as an early access in Kafka version 4.0 which is supported by Strimzi 0.46.  In this blog, I’m going to introduce this feature and how you can try it out with your Strimzi managed cluster. 
```

Comment:
It might be good to mention the early access limitations? IIRC it has also some upgrade impact which definitely should be mentioned with a warning to not use this in production?

Reply:
TBH, I would not hold it - or to be clear, I do not have problem if others want to wait with it, but I see no need for it. Just make it more clear what the state and the plan is.

Even if it is production ready in 4.1 and we wait for Strimzi 0.48, we would still need to mention it as the blog post is not directly related to any particular release. So one way or another, this will need to be covered even if we wait.

Reply:
I have left it for now, as I would need to try it out once 4.1 is released and is supported in Strimzi. If that happens before posting this blog (which might be the case anyway), I will update it.